### DISCUSSION TITLE
How to make more options for the select field of the table field in Freeform Pro

### DISCUSSION THREAD

**Discussion by stephano41** [solspace/craft-freeform#211] [Questions & Answers] [answered]
Hi, I am trying to make a table inside a form from Freeform Pro, and one of the field types inside the table is a "select" field, and I can't figure out how to add more than one option to the select field of a table field. Particularly, when I type a value into the value column, it shows up as an option, but I can't figure out how I can type multiple words so that they show up as different options.
![image](https://user-images.githubusercontent.com/78075921/136885808-f97bab00-d918-41a9-b776-21b9123e5025.png)
![image](https://user-images.githubusercontent.com/78075921/136885820-658bc21c-b493-4b13-a875-8479072f141a.png)

Any help on how to add more options would be appreciated!

**Comment by kjmartens** [comment, marked as answer]
We'll have an improved interface eventually, but for now, to have more than 1 option value you just need to separate values with the `;` character, e.g. `Option 1;Option 2;Option 3`. 🙂
